about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Elite school website</title>
+  <link rel="stylesheet" href="css/about.css">
+
+  <!-------------------------swiper js cocogoose font----------------------------->
+  <link rel="stylesheet" href="https://unpkg.com/swiper@8/swiper-bundle.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&family=Roboto+Flex:opsz,wght@8..144,400;8..144,500;8..144,600&display=swap"
+    rel="stylesheet">
+  <!-------------------------swiper js----------------------------->
+
+</head>
+
+<body>
+  <header>
```

Comment:
- [ ] Kindly add the top dark nav to your header as this is part of the project requirements 👍🏿 

![Screenshot_206](https://user-images.githubusercontent.com/69638013/169273428-078c1434-db20-43e6-b126-d282fd5b4923.png)